packages/devtools/client/components/SideNav.vue

```diff
@@ -83,7 +83,7 @@ onClickOutside(
         >
           <div i-logos-nuxt-icon h-6 w-6 />
           <template v-if="sidebarExpanded">
-            <span text="lg white" font-600>
+            <span text="lg black dark:white" font-600>
```

Comment:
```suggestion
            <span text-lg font-600>
```

Would this works, that just inherit the parent color?

Reply:
it would work but it's not white or black, it's kinda gray

<img width="252" alt="Screenshot 2023-12-08 at 1 26 41 PM" src="https://github.com/nuxt/devtools/assets/38922203/5fff1db8-5422-4d78-8013-e00685a3c3d2">
<img width="247" alt="Screenshot 2023-12-08 at 1 27 08 PM" src="https://github.com/nuxt/devtools/assets/38922203/92fc7952-b635-4048-b3c1-491bff604dfb">

Reply:
No it was great before this, thanks!

Reply:
Thank you